### DISCUSSION TITLE
Please help me understand some variables: Center_bias and Class_temperature

### DISCUSSION THREAD

**Discussion by LtqxWYEG** [lucidrains/big-sleep#74] [Q&A]
Hi :)

Can someone please explain to me, _roughly_, what **"Class_temperature"** and **"Center_bias"** does?
**Class_temperature** sounds to me like a modificator/factor/the gravity of CLEAR-classes found in the TEXT string.
**Center_bias** sounds like it's a toggle for encouraging more detail in the center of the image.

Is any of that correct?

Thank you for your help! :)

**Comment by jaazza** [comment]
I've been looking for the info on the variables as well. Seems hard to come by. Hopefully someone will pop in here with some insight.
